Documentation/GeneralConventions/FileStructure.rst

```diff
@@ -47,129 +47,27 @@ Further conventions are:
 Full documentation
 ==================
 
-This is the recommended setup and is commonly used in the official TYPO3 manuals
-and extension documentation.
+To render a complete documentation manual you need a folder called
+:file:`Documentation` with at least an entry point reStrutured Text file called
+:file:`Documentation\Index.rst` and a configuration file called
+:file:`Documentation\guides.xml`. Add more files as needed
 
-This structure splits the documentation for the VCS host (README.rst) and the
-Sphinx theme (Documentation/) and allows full use of a continuously expanded set
-of :ref:`custom content elements <rest-reference>`, like UML diagrams, which are
-only supported by the theme. This structure allows to add multiple pages to
-the documentation and build a full page tree.
-
-The Settings.cfg configuration file allows you to set theme variables, i.e. the
-project title, release version and the like.
-
-Of course, you can also use a README.md instead of a README.rst file as both
-markup languages are supported by the common VCS hosts.
+You can keep :file:`a README.md` or :file:`README.rst` file with basic
+information and a link to the published manual. These files will be commonly
+displayed on GitHub and GitLab.
 
 .. code-block:: none
 
    .
+   ├── composer.json
    ├── README.rst
    └── Documentation
-       ├── genindex.rst
-       ├── Includes.rst.txt
+       ├── About.rst
```

Comment:
Why `About.rst` and what is the purpose of it? By default, we have an `Introduction/Index.rst` file.

I suggest to stick to the example extension manual.

Reply:
removed it.